pkg/cmd/server/apis/config/v1/testdata/master-config.yaml

```diff
@@ -121,6 +121,7 @@ networkConfig:
   ingressIPNetworkCIDR: 172.29.0.0/16
   networkPluginName: ""
   serviceNetworkCIDR: 10.0.0.0/24
+  vxlanPort: 4789
```

Comment:
Do we end up testing anywhere that a master-config with no vxlanPort specified will be interpreted correctly?

Reply:
I have manually tested but I am not sure how to test what you are suggesting. This test makes sure that the given config matches one that goes through the serializer